packages/vitest/package.json

```diff
@@ -34,5 +34,8 @@
   "devDependencies": {
     "effect": "workspace:^",
     "vitest": "^2.1.4"
+  },
+  "dependencies": {
+    "fast-check": "^3.23.1"
```

Comment:
This would have to be a peerDependency

Reply:
Cool. Do I need to add changset for this?

Reply:
done

Reply:
also pushed a changeset

Reply:
Wait a second... `fast-check` is already a dependency of `effect`. So this should not be needed altogether.

Instead, we might want to import `FastCheck` from `effect` within `vitest`.

E.g.

```ts
import * as FastCheck from "effect/FastCheck"
```

Reply:
Good catch, that’s preferable